contracts/oracle-verifier/src/msg.rs

```diff
@@ -42,17 +37,17 @@ pub enum ExecuteMsg {
 #[cw_orch(disable_fields_sorting)]
 #[derive(QueryResponses)]
 pub enum QueryMsg {
-    #[returns(Config)]
+    #[returns(crate::state::Config)]
     Config {},
     /// Ordered by completion time descending (last completed first)
-    #[returns(Option<TaskInfoResponse>)]
+    #[returns(Option<lavs_apis::verifier_simple::TaskInfoResponse>)]
```

Comment:
Can you explain why these changes are needed?

Reply:
funny timing, I think I was writing on Slack literally as you were writing here :)

it's not really gui-specific, it's a warning generated whenever using the contract as a library, for its types, because then it's only used as a schema and not referenced anywhere else so it generates "unused variable" warnings